src/utils/hooks/useLoadApplication.ts

```diff
@@ -99,15 +107,14 @@ const useLoadApplication = ({ serialNumber, networkFetch }: UseGetApplicationPro
       serial: application.serial as string,
       name: application.name as string,
       outcome: application.outcome as string,
-      current: {
-        stage: {
-          id: stageId as number,
-          name: stage as string,
-          number: stageNumber as number,
-          colour: stageColour as string,
-        },
+      currentStage: {
+        id: stageId as number,
+        name: stage as string,
+        number: stageNumber as number,
+        colour: stageColour as string,
+        createdDate: statusHistoryTimeCreated,
         status: status as ApplicationStatus,
-        date: statusHistoryTimeCreated,
+        statusCreatedDate: stageHistoryTimeCreated,
```

Comment:
is this correct ? there is no graphQL query for this and stage history time created relates to creation time of stage not latest status. similar to comment about review, maybe we should seperate currentStage state from currentStatus state

Reply:
Good catch, this is wrong. It should be the opposite way. Changing this now

Reply:
What about the graphQL part of the comment? I can't see where it's queried

Reply:
Oh, I didn't see this one. Thanks! Fixing now : )